Vagrantfile

```diff
@@ -0,0 +1,131 @@
+# -*- mode: ruby -*-
```

Comment:
See comments on other projects re vagrant files - purpose and expected fate of this file should be documented